BalancedBinaryTree.java

```diff
@@ -0,0 +1,30 @@
+// 110. Balanced Binary Tree
+// https://leetcode.com/problems/balanced-binary-tree/
+
+//The time complexity is O(n), because we may traverse the whole tree.
+//Space complexity: o(N) The recursion stack may go up to O(n) if the tree is unbalanced.
```

Comment:
Worst case will be N but it is better to represent as O(h) h-> height of the tree